nix/modules/tezos-node.nix

```diff
@@ -72,7 +71,19 @@ let
 
       nodeConfig = mkOption {
         default = null;
-        type = types.nullOr (import sources.serokell-nix).lib.types.jsonConfig;
```

Comment:
Were these types removed from `serokell-nix`?

Reply:
I failed to understand how to pass argument to nixos module, actually I just copied definition so it works fine till you help with proper solution